rocAL/rocAL/source/augmentations/geometry_augmentations/node_resize.cpp

```diff
@@ -60,14 +60,24 @@ void ResizeNode::create_node() {
 
 void ResizeNode::update_node() {
     std::vector<uint32_t> src_h_dims, src_w_dims;
+    std::vector<uint32_t> original_h_dims, original_w_dims;
     src_w_dims = _inputs[0]->info().get_roi_width_vec();
     src_h_dims = _inputs[0]->info().get_roi_height_vec();
+    original_w_dims = _inputs[0]->info().get_original_roi_width_vec();
```

Comment:
We could have an if condition to check if it is resize shorter, then we can assign original roi width and height to the src_w_dims and src_h_dims and then call the adjust_out_roi_size function to maintain uniformity

Also add a comment

Reply:
Resolved in the latest commit. Will trigger a training to confirm convergence. Slightly doubtful whether std::lround in adjust_roi_size may cause difference in the resize dimensions.